terraform/aws/k8s_manifests.tf

```diff
@@ -0,0 +1,69 @@
+provider "kubernetes" {
+  config_path    = "~/.kube/config"
+}
```

Comment:
This presents a craptacular problem b/c the credentials have to exist for the cluster we are creating before it can talk to the cluster. 

There might be a way to script this.

 Don't see many better options herez: https://registry.terraform.io/providers/hashicorp/kubernetes/latest/docs/guides/getting-started#provider-setup

Reply:
Should be able to do something like what we do with the helm provider, no? https://github.com/pangeo-forge/pangeo-forge-cloud-federation/blob/5e98329598419089eecc37c2a9feb8750299c72b/terraform/aws/support.tf#L1

See https://github.com/2i2c-org/infrastructure/blob/49e863594b19542f644130875317924a5f0cb42e/terraform/gcp/main.tf#L55 for an example of something similar for the kubernetes provider. I think something should exist like that here too?

And terraform orders these dependencies correctly.

Reply:
doh, I shoud've been able to see that myself 🦃

Reply:
solve this